### PR TITLE
Hammer/fix uat

### PR DESCRIPTION
**Pull request by aaronchongth**
## What's new

- [x] More logs for schedule times and conversions
- [x] using `datetime.fromtimestamp(self.until.timestamp()))`
- [x] test with changed pod timezone
- [x] export csv to designate semicolons as delimiters for Excel
- [x] fix schedule glitching due to logging
- [x] change logs to debug logs instead
- [x] disables unused and confusing start time field
- [x] lift op mode to mode, from lift state
- [x] additional schedule event viewer component for entire task description

## Self-checks

- [ ] I have prototyped this new feature (if necessary) on Figma 
- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: packages/api-server/api_server/models/tortoise_models/scheduled_task.py [hunk 1]
```diff
@@ -71,7 +72,15 @@ def to_job(self) -> Job:
             # schedule uses `datetime.now()`, which is tz naive
             # Assuming self.until is a datetime object with timezone information
             # Convert the timestamp to datetime without changing the timezone
-            job = job.until(datetime.utcfromtimestamp(self.until.timestamp()))
+            print(f"to_job until: {self.until}")
+            print(f"to_job until timestamp: {self.until.timestamp()}")
+            print(
+                f"to_job until utcfromtimestamp: {datetime.utcfromtimestamp(self.until.timestamp())}"
+            )
```
Comment (koonpeng) [inline packages/api-server/api_server/models/tortoise_models/scheduled_task.py:79]: Don't think we need to print this.

File: packages/api-server/setup.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
         "tortoise-orm~=0.18.1",
         "pyjwt[crypto]~=2.4",
         "pydantic~=1.9",
-        "schedule~=1.1.0",
+        "schedule~=1.2.1",
```
Comment (koonpeng) [inline packages/api-server/setup.py:29]: Is the timezone feature not available in 1.1.0?

File: packages/dashboard/src/components/appbar.tsx [hunk 3]
```diff
@@ -229,15 +229,21 @@ export const AppBar = React.memo(({ extraToolbarItems }: AppBarProps): React.Rea
       }
       if (!schedule) {
         await Promise.all(
-          taskRequests.map((request) =>
-            rmf.tasksApi.postDispatchTaskTasksDispatchTaskPost({
+          taskRequests.map((request) => {
+            console.log(`submitTask: ${request}`);
```
Comment (koonpeng) [inline packages/dashboard/src/components/appbar.tsx:233]: These `console.log` could be replaced with `debug`.

File: packages/api-server/setup.py (hunk 2 above)
Comment (aaronchongth) [reply packages/api-server/setup.py:29]: nope, unfortunately, it was in 1.2.0, but 1.2.1 comes with some bug fixes.
https://github.com/dbader/schedule/pull/576
https://github.com/dbader/schedule/pull/600

File: packages/dashboard/src/components/appbar.tsx (hunk 3 above)
Comment (aaronchongth) [reply packages/dashboard/src/components/appbar.tsx:233]: I'll clean them up as debug messages after the demo then

**Review by koonpeng — APPROVED** [review]

**Pull request merged**